public/src/components/channelManagement/bannerDesigns/LiveSwitch.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
would it be possible to reuse this one?
https://github.com/guardian/support-admin-console/blob/main/public/src/components/shared/liveSwitch.tsx

Reply:
Oh ha! I hadn't spotted that one. This code was adapted from the test live switch (which has modal behaviour I didn't want, hence not using it). But yeah that looks identical, thanks I'll update.

Reply:
👍  Fixed in 4b9fba582b9c4bd3495cae97fd79d53277131f3d.